.github/workflows/Windows.yml

```
@@ -28,10 +28,10 @@ jobs:
   Windows:
     runs-on: windows-2022
     env:
-      PRESET: ${{inputs.preset || 'win-x64-rel'}}
-      VCPKG_DEFAULT_TRIPLET: ${{inputs.preset == 'win-x86-rel' && 'x86-windows' || 'x64-windows-release'}}
+      PRESET: ${{inputs.preset || 'win-x64-static'}}
+      VCPKG_DEFAULT_TRIPLET: ${{inputs.preset == 'win-x86-static' && 'x86-windows' || 'x64-windows-staticease'}}
```

```suggestion
      VCPKG_DEFAULT_TRIPLET: ${{inputs.preset == 'win-x86-static' && 'x86-windows' || 'x64-windows-static-release'}}
```
I assume you meant that?